FluentUI.Demo/src/androidTest/java/com/microsoft/fluentuidemo/demos/V2PeoplePickerUITest.kt

```diff
@@ -141,7 +146,8 @@ class V2PeoplePickerUITest {
         val textField = composeTestRule.onNodeWithTag("V2PeoplePicker")
         textField.assertExists()
         textField.assertIsDisplayed()
-        val peoplePickerChip = composeTestRule.onNodeWithText("firstName", true, useUnmergedTree = true)
+        val peoplePickerChip =
```

Comment:
space formatting

Reply:
Formatting file change. not added intentionally.